### PR TITLE
Add apostrophes to contractions

### PR DESCRIPTION
**Pull request by alexdmiller**
I found the text very hard to read due to the lack of apostrophes to differentiate `it's` from `its`. This PR picks off the low hanging fruit via find-and-replace. [Here's](https://www.dummies.com/education/language-arts/grammar/how-to-use-apostrophes-to-form-contractions/) a guide to contractions in English which might be useful.

### CODE REVIEW

File: docs/chapter-1/vulkan_renderpass_code.md [hunk 1]
```diff
@@ -72,8 +72,8 @@ void VulkanEngine::init_default_renderpass()
 	// we keep the attachment stored when the renderpass ends
 	color_attachment.storeOp = VK_ATTACHMENT_STORE_OP_STORE;
 	//we don't care about stencil
-	color_attachment.stencilLoadOp = VK_ATTACHMENT_LOAD_OP_DONT_CARE;	
-	color_attachment.stencilStoreOp = VK_ATTACHMENT_STORE_OP_DONT_CARE;
+	color_attachment.stencilLoadOp = VK_ATTACHMENT_LOAD_OP_don't_CARE;
```
Comment (cdgiessen) [inline docs/chapter-1/vulkan_renderpass_code.md:75]: If this is due to a search and replace gone wrong, then it needs to be corrected.
Even the Vulkan API gets it 'wrong' it seems. (Though I would not want to have to type grammatically correct English for what should be a programming API...)

File: docs/chapter-3/depth_buffer.md [hunk 2]
```diff
@@ -204,7 +204,7 @@ void VulkanEngine::init_default_renderpass()
     depth_attachment.loadOp = VK_ATTACHMENT_LOAD_OP_CLEAR;
     depth_attachment.storeOp = VK_ATTACHMENT_STORE_OP_STORE;
     depth_attachment.stencilLoadOp = VK_ATTACHMENT_LOAD_OP_CLEAR;
-    depth_attachment.stencilStoreOp = VK_ATTACHMENT_STORE_OP_DONT_CARE;
+    depth_attachment.stencilStoreOp = VK_ATTACHMENT_STORE_OP_don't_CARE;
```
Comment (cdgiessen) [inline docs/chapter-3/depth_buffer.md:207]: Same as above.

**Review by cdgiessen — CHANGES_REQUESTED** [review]
All but 2 places look okay to me.

**Comment by vblanco20-1** [comment]
All good, thanks for the help

**Pull request merged**